code/game/gamemodes/cult/runes.dm

```diff
@@ -332,10 +332,12 @@ var/list/teleport_runes = list()
 			user.forceMove(get_turf(actual_selected_rune))
 		var/mob/living/carbon/human/H = user
 		if(user.z != T.z)
-			H.bleed(5)
+			if(istype(H)) 
```

Comment:
Why not just return, above, if it's not human? This doesn't seem to be something anything other than a human can do anyway.

Reply:
Constructs can also do this. Or they should be able to do this. Currently they cannot because it runtimes on them